build/updater.sh

```diff
@@ -2,6 +2,7 @@
 cd query-builder
 git pull
 rm -rf node_modules
+rm package-lock.json
```

Comment:
We shouldn't do this. We [intentionally check in the package-lock.json](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5) to ensure that our different environments (dev, test system, prod) install the same dependencies. Removing it defeats the purpose of checking it in. If the NPM version is a problem, can't we just install the right one within the container? This is [what our Dockerfile does](https://github.com/wmde/query-builder/blob/master/Dockerfile#L14), too.

Reply:
Oh, It errored when installing and google said to delete the file. It actually makes a lot of sense. Let me give it a try.

Reply:
Tested and it worked just fine. Changed now. Thanks for the hint.